### ISSUE TITLE
Only get a blue screen

### ISSUE THREAD

**Issue by TheLastProject** [snapcrafters/tmnationsforever#8]
I assume this is supposed to be the installer, but I only get a blue screen that seems to stay forever (I waited several minutes).

![screenshot_20180616_233603](https://user-images.githubusercontent.com/1885159/41502604-29419072-71be-11e8-833a-75b7f0493dda.png)

I'm on Fedora 28 with KDE Plasma 5.

I notice the following in my log:
```
$ tmnationsforever                                                                                                                                      
Fontconfig warning: line 5: Fontconfig warning: line 5: unknown element "its:rules"unknown element "its:rules"                                                              
                                                                                                                                                                            
Fontconfig warning: line 6: Fontconfig warning: line 6: unknown element "its:translateRule"unknown element "its:translateRule"                                              
                                                                                                                                                                            
Fontconfig error: line 6: invalid attribute 'translate'Fontconfig error: line 6:                                                                                            
invalid attribute 'translate'Fontconfig error: line 6:                                                                                                                      
invalid attribute 'selector'Fontconfig error: line 6:                                                                                                                       
invalid attribute 'selector'                                                                                                                                                
Fontconfig error: line 7: invalid attribute 'xmlns:its'Fontconfig error: line 7:                                                                                            
invalid attribute 'xmlns:its'Fontconfig error: line 7:                                                                                                                      
invalid attribute 'version'Fontconfig error: line 7:                                                                                                                        
invalid attribute 'version'                                                                                                                                                 
Fontconfig warning: line 9: unknown element "description"Fontconfig warning: line 9:                                                                                        
unknown element "description"                                                                                                                                               
Fontconfig error: Cannot load default config file                                                                                                                           
Fontconfig error: Cannot load default config file                                                                                                                           
Gtk-Message: GtkDialog mapped without a transient parent. This is discouraged.                                                                                              
Gtk-Message: GtkDialog mapped without a transient parent. This is discouraged.                                                                                              
Fontconfig warning: line 5: unknown element "its:rules"                                                                                                                     
Fontconfig warning: line 6: unknown element "its:translateRule"                                                                                                             
Fontconfig error: line 6: invalid attribute 'translate'                                                                                                                     
Fontconfig error: line 6: invalid attribute 'selector'                                                                                                                      
Fontconfig error: line 7: invalid attribute 'xmlns:its'                                                                                                                     
Fontconfig error: line 7: invalid attribute 'version'                                                                                                                       
Fontconfig warning: line 9: unknown element "description"                                                                                                                   
Fontconfig error: Cannot load default config file                                                                                                                           
Gtk-Message: GtkDialog mapped without a transient parent. This is discouraged.                                                                                              
Fontconfig warning: line 5: unknown element "its:rules"                                                                                                                     
Fontconfig warning: line 6: unknown element "its:translateRule"                                                                                                             
Fontconfig error: line 6: invalid attribute 'translate'                                                                                                                     
Fontconfig error: line 6: invalid attribute 'selector'                                                                                                                      
Fontconfig error: line 7: invalid attribute 'xmlns:its'                                                                                                                     
Fontconfig error: line 7: invalid attribute 'version'                                                                                                                       
Fontconfig warning: line 9: unknown element "description"                                                                                                                   
Fontconfig error: Cannot load default config file                                                                                                                           
Fontconfig warning: line 5: unknown element "its:rules"                                                                                                                     
Fontconfig warning: line 6: unknown element "its:translateRule"                                                                                                             
Fontconfig error: line 6: invalid attribute 'translate'                                                                                                                     
Fontconfig error: line 6: invalid attribute 'selector'                                                                                                                      
Fontconfig error: line 7: invalid attribute 'xmlns:its'                                                                                                                     
Fontconfig error: line 7: invalid attribute 'version'                                                                                                                       
Fontconfig warning: line 9: unknown element "description"                                                                                                                   
Fontconfig error: Cannot load default config file
0025:err:ntdll:RtlpWaitForCriticalSection section 0x7bce8b60 "loader.c: loader_section" wait timed out in thread 0025, blocked by 0026, retrying (60 sec)                  
000f:err:service:process_send_command receiving command result timed out
000f:fixme:service:scmdatabase_autostart_services Auto-start service L"WineBus" failed to start: 1053                                                                      
Fontconfig warning: line 5: unknown element "its:rules"
Fontconfig warning: line 6: unknown element "its:translateRule"
Fontconfig error: line 6: invalid attribute 'translate'
Fontconfig error: line 6: invalid attribute 'selector'
Fontconfig error: line 7: invalid attribute 'xmlns:its'
Fontconfig error: line 7: invalid attribute 'version'
Fontconfig warning: line 9: unknown element "description"
Fontconfig error: Cannot load default config file
Fontconfig warning: line 5: unknown element "its:rules"
Fontconfig warning: line 6: unknown element "its:translateRule"
Fontconfig error: line 6: invalid attribute 'translate'
Fontconfig error: line 6: invalid attribute 'selector'
Fontconfig error: line 7: invalid attribute 'xmlns:its'
Fontconfig error: line 7: invalid attribute 'version'
Fontconfig warning: line 9: unknown element "description"
Fontconfig error: Cannot load default config file
Fontconfig warning: line 5: unknown element "its:rules"
Fontconfig warning: line 6: unknown element "its:translateRule"
Fontconfig error: line 6: invalid attribute 'translate'
Fontconfig error: line 6: invalid attribute 'selector'
Fontconfig error: line 7: invalid attribute 'xmlns:its'
Fontconfig error: line 7: invalid attribute 'version'
Fontconfig warning: line 9: unknown element "description"
Fontconfig error: Cannot load default config file
0028:err:winediag:nodrv_CreateWindow Application tried to create a window, but no driver could be loaded.
0028:err:winediag:nodrv_CreateWindow The explorer process failed to start.
0028:err:ole:apartment_createwindowifneeded CreateWindow failed with error 0
0028:err:ole:apartment_createwindowifneeded CreateWindow failed with error 0
0028:err:ole:apartment_createwindowifneeded CreateWindow failed with error 0
Fontconfig warning: line 5: unknown element "its:rules"
Fontconfig warning: line 6: unknown element "its:translateRule"
Fontconfig error: line 6: invalid attribute 'translate'
Fontconfig error: line 6: invalid attribute 'selector'
Fontconfig error: line 7: invalid attribute 'xmlns:its'
Fontconfig error: line 7: invalid attribute 'version'
Fontconfig warning: line 9: unknown element "description"
Fontconfig error: Cannot load default config file
0009:err:winediag:nodrv_CreateWindow Application tried to create a window, but no driver could be loaded.
0009:err:winediag:nodrv_CreateWindow The explorer process failed to start.
removed '/tmp/tmnationsforever_setup.exe'
/snap/tmnationsforever/16/bin/sommelier: line 40: cd: /home/sylvia/snap/tmnationsforever/common/.wine/drive_c/Program Files/TmNationsForever: No such file or directory
Fontconfig warning: line 5: unknown element "its:rules"
Fontconfig warning: line 6: unknown element "its:translateRule"
Fontconfig error: line 6: invalid attribute 'translate'
Fontconfig error: line 6: invalid attribute 'selector'
Fontconfig error: line 7: invalid attribute 'xmlns:its'
Fontconfig error: line 7: invalid attribute 'version'
Fontconfig warning: line 9: unknown element "description"
Fontconfig error: Cannot load default config file
002f:err:winediag:nodrv_CreateWindow Application tried to create a window, but no driver could be loaded.
002f:err:winediag:nodrv_CreateWindow The explorer process failed to start.
002f:err:ole:apartment_createwindowifneeded CreateWindow failed with error 0
002f:err:ole:apartment_createwindowifneeded CreateWindow failed with error 0
002f:err:ole:apartment_createwindowifneeded CreateWindow failed with error 0
Fontconfig warning: line 5: unknown element "its:rules"
Fontconfig warning: line 6: unknown element "its:translateRule"
Fontconfig error: line 6: invalid attribute 'translate'
Fontconfig error: line 6: invalid attribute 'selector'
Fontconfig error: line 7: invalid attribute 'xmlns:its'
Fontconfig error: line 7: invalid attribute 'version'
Fontconfig warning: line 9: unknown element "description"
Fontconfig error: Cannot load default config file
Fontconfig warning: line 5: unknown element "its:rules"
Fontconfig warning: line 6: unknown element "its:translateRule"
Fontconfig error: line 6: invalid attribute 'translate'
Fontconfig error: line 6: invalid attribute 'selector'
Fontconfig error: line 7: invalid attribute 'xmlns:its'
Fontconfig error: line 7: invalid attribute 'version'
Fontconfig warning: line 9: unknown element "description"
Fontconfig error: Cannot load default config file
Fontconfig warning: line 5: unknown element "its:rules"
Fontconfig warning: line 6: unknown element "its:translateRule"
Fontconfig error: line 6: invalid attribute 'translate'
Fontconfig error: line 6: invalid attribute 'selector'
Fontconfig error: line 7: invalid attribute 'xmlns:its'
Fontconfig error: line 7: invalid attribute 'version'
Fontconfig warning: line 9: unknown element "description"
Fontconfig error: Cannot load default config file
```

**Comment by hyphone** [comment]
Same here with Fedora 28 GNOME on Wayland

**Comment by Danie10** [comment]
I get similar errors with Manjaro Arch Linux KDE. See a small popup window saying "Preparing Windows environment".

**Comment by Ads20000** [comment]
Could we have the outputs of `snap version`, the output of the `refreshed`, `tracking`, and `installed` lines of `snap info tmnationsforever` and `snap info core` please? :)

**Comment by hyphone** [comment]
sure :)

Fedora 28

**snap version**
```
snap    2.32.4-1.fc28
snapd   2.32.4-1.fc28
series  16
fedora  28
kernel  4.17.2-200.fc28.x86_64
```

**snap info tmnationsforever**
```
tracking:  stable
refreshed: 2018-06-15T02:26:21+02:00
installed:   2.11.26 (16) 129MB
```

**snap info core**
```
tracking:  stable
refreshed: 2018-06-08T18:15:10+02:00
installed:   16-2.33                  (4830) 91MB core
```

**Comment by BradleyNorris777** [comment]
Recently attempted install and got this problem as well on Ubuntu 18.10

**Snap Version**
snap    2.36.3
snapd   2.36.3
series  16
ubuntu  18.10
kernel  4.18.0-13-generic

**TMNationsForever**
tracking:     stable
refresh-date: yesterday at 18:47 CST
installed:   2.11.26 (26) 131MB -

**Info Core**
refresh-date: 9 days ago, at 18:23 CST
tracking:     stable
installed:   16-2.36.3                 (6130) 93MB core

**Comment by Ads20000** [comment]
Hmm, it would be good if someone with more expertise than me (I'm not a dev so don't know how to help much with the actual snaps) could adjust the `edge` snap to launch in Terminal so we could find out what's happening.

**Comment by Ads20000** [comment]
What's the Terminal output when you run `snap run tmnationsforever`?

**Comment by Ads20000** [comment]
@flexiondotorg this issue came up two days after the last Winetricks update (https://github.com/snapcrafters/tmnationsforever/commit/e77a987702da04d8ec0f885293cd2a10c0b5a542), do you think updating Winetricks to latest (https://github.com/Winetricks/winetricks/releases/tag/20181203) or reverting to https://github.com/Winetricks/winetricks/releases/tag/20180513 to test would help?

I wonder this because the Terminal output given in the original post implies potentially a problem with `corefonts`, unless the Terminal output fails to identify the cause and actually people without this Issue have those errors in Terminal, and I suppose Winetricks handles `corefonts`...

Would `journalctl` give any useful output in this case? I can't imagine Wine interacts much with `systemd`?

**Comment by black2996** [comment]
the same here on Cosmic:
snap    2.37.2
snapd   2.37.2
series  16
kernel  4.18.0-15-generic
tmnf 2.11.26 stable

**Comment by tgrzesik** [comment]
the same problem:
snapd: 2.37.4
system: fedora 29
kernel: .20.13-200.fc29.x86_64
tmnf: 2.11.26

**Comment by xordspar0** [comment]
I kind of doubt that the fonts have anything to do with it. I'm more interested in this line in the output:

```
/snap/tmnationsforever/x1/bin/sommelier: line 40: cd: /home/jordan/snap/tmnationsforever/common/.wine/drive_c/Program Files/TmNationsForever: No such file or directory
```

Based on what my `~/snap/tmnationsforever` directory like, the script should be looking in `common/`, not `current/`.

**Comment by xordspar0** [comment]
@Ads20000 I just tried updating to the latest version of winetricks. No change.

**Comment by maciejlagowski** [comment]
same here with Fedora 31 KDE
kernel  5.3.16-300.fc31.x86_64

"[maciek@localhost ~]$ optirun tmnationsforever
ERROR: ld.so: object 'libdlfaker.so' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
ERROR: ld.so: object 'libvglfaker.so' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
WARNING: cgroup v2 is not fully supported yet, proceeding with partial confinement
Fontconfig warning: line 5: unknown element "its:rules"
Fontconfig warning: line 6: unknown element "its:translateRule"
Fontconfig error: line 6: invalid attribute 'translate'
Fontconfig error: line 6: invalid attribute 'selector'
Fontconfig error: line 7: invalid attribute 'xmlns:its'
Fontconfig error: line 7: invalid attribute 'version'
Fontconfig warning: line 9: unknown element "description"
Fontconfig error: Cannot load default config file
Gtk-Message: GtkDialog mapped without a transient parent. This is discouraged.
Fontconfig warning: line 5: unknown element "its:rules"
Fontconfig warning: line 6: unknown element "its:translateRule"
Fontconfig error: line 6: invalid attribute 'translate'
Fontconfig error: line 6: invalid attribute 'selector'
Fontconfig error: line 7: invalid attribute 'xmlns:its'
Fontconfig error: line 7: invalid attribute 'version'
Fontconfig warning: line 9: unknown element "description"
Fontconfig error: Cannot load default config file
Fontconfig warning: line 5: unknown element "its:rules"
Fontconfig warning: line 6: unknown element "its:translateRule"
Fontconfig error: line 6: invalid attribute 'translate'
Fontconfig error: line 6: invalid attribute 'selector'
Fontconfig error: line 7: invalid attribute 'xmlns:its'
Fontconfig error: line 7: invalid attribute 'version'
Fontconfig warning: line 9: unknown element "description"
Fontconfig error: Cannot load default config file
000f:err:service:process_send_command receiving command result timed out
000f:fixme:service:scmdatabase_autostart_services Auto-start service L"WineBus" failed to start: 1053
Fontconfig warning: line 5: unknown element "its:rules"
Fontconfig warning: line 6: unknown element "its:translateRule"
Fontconfig error: line 6: invalid attribute 'translate'
Fontconfig error: line 6: invalid attribute 'selector'
Fontconfig error: line 7: invalid attribute 'xmlns:its'
Fontconfig error: line 7: invalid attribute 'version'
Fontconfig warning: line 9: unknown element "description"
Fontconfig error: Cannot load default config file
Fontconfig warning: line 5: unknown element "its:rules"
Fontconfig warning: line 6: unknown element "its:translateRule"
Fontconfig error: line 6: invalid attribute 'translate'
Fontconfig error: line 6: invalid attribute 'selector'
Fontconfig error: line 7: invalid attribute 'xmlns:its'
Fontconfig error: line 7: invalid attribute 'version'
Fontconfig warning: line 9: unknown element "description"
Fontconfig error: Cannot load default config file
Fontconfig warning: line 5: unknown element "its:rules"
Fontconfig warning: line 6: unknown element "its:translateRule"
Fontconfig error: line 6: invalid attribute 'translate'
Fontconfig error: line 6: invalid attribute 'selector'
Fontconfig error: line 7: invalid attribute 'xmlns:its'
Fontconfig error: line 7: invalid attribute 'version'
Fontconfig warning: line 9: unknown element "description"
Fontconfig error: Cannot load default config file
0027:err:winediag:nodrv_CreateWindow Application tried to create a window, but no driver could be loaded.
0027:err:winediag:nodrv_CreateWindow The explorer process failed to start.
0027:err:ole:apartment_createwindowifneeded CreateWindow failed with error 0
0027:err:ole:apartment_createwindowifneeded CreateWindow failed with error 0
0027:err:ole:apartment_createwindowifneeded CreateWindow failed with error 0
Fontconfig warning: line 5: unknown element "its:rules"
Fontconfig warning: line 6: unknown element "its:translateRule"
Fontconfig error: line 6: invalid attribute 'translate'
Fontconfig error: line 6: invalid attribute 'selector'
Fontconfig error: line 7: invalid attribute 'xmlns:its'
Fontconfig error: line 7: invalid attribute 'version'
Fontconfig warning: line 9: unknown element "description"
Fontconfig error: Cannot load default config file
0009:err:winediag:nodrv_CreateWindow Application tried to create a window, but no driver could be loaded.
0009:err:winediag:nodrv_CreateWindow The explorer process failed to start.
removed '/tmp/tmnationsforever_setup.exe'
/snap/tmnationsforever/32/bin/sommelier: line 40: cd: /home/maciek/snap/tmnationsforever/common/.wine/drive_c/Program Files/TmNationsForever: No such file or directory
Fontconfig warning: line 5: unknown element "its:rules"
Fontconfig warning: line 6: unknown element "its:translateRule"
Fontconfig error: line 6: invalid attribute 'translate'
Fontconfig error: line 6: invalid attribute 'selector'
Fontconfig error: line 7: invalid attribute 'xmlns:its'
Fontconfig error: line 7: invalid attribute 'version'
Fontconfig warning: line 9: unknown element "description"
Fontconfig error: Cannot load default config file
002b:err:winediag:nodrv_CreateWindow Application tried to create a window, but no driver could be loaded.
002b:err:winediag:nodrv_CreateWindow The explorer process failed to start.
002b:err:ole:apartment_createwindowifneeded CreateWindow failed with error 0
002b:err:ole:apartment_createwindowifneeded CreateWindow failed with error 0
002b:err:ole:apartment_createwindowifneeded CreateWindow failed with error 0
[maciek@localhost ~]$ optirun tmnationsforever
ERROR: ld.so: object 'libdlfaker.so' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
ERROR: ld.so: object 'libvglfaker.so' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
WARNING: cgroup v2 is not fully supported yet, proceeding with partial confinement
Fontconfig warning: line 5: unknown element "its:rules"
Fontconfig warning: line 6: unknown element "its:translateRule"
Fontconfig error: line 6: invalid attribute 'translate'
Fontconfig error: line 6: invalid attribute 'selector'
Fontconfig error: line 7: invalid attribute 'xmlns:its'
Fontconfig error: line 7: invalid attribute 'version'
Fontconfig warning: line 9: unknown element "description"
Fontconfig error: Cannot load default config file
Gtk-Message: GtkDialog mapped without a transient parent. This is discouraged.
Fontconfig warning: line 5: unknown element "its:rules"
Fontconfig warning: line 6: unknown element "its:translateRule"
Fontconfig error: line 6: invalid attribute 'translate'
Fontconfig error: line 6: invalid attribute 'selector'
Fontconfig error: line 7: invalid attribute 'xmlns:its'
Fontconfig error: line 7: invalid attribute 'version'
Fontconfig warning: line 9: unknown element "description"
Fontconfig error: Cannot load default config file
Fontconfig warning: line 5: unknown element "its:rules"
Fontconfig warning: line 6: unknown element "its:translateRule"
Fontconfig error: line 6: invalid attribute 'translate'
Fontconfig error: line 6: invalid attribute 'selector'
Fontconfig error: line 7: invalid attribute 'xmlns:its'
Fontconfig error: line 7: invalid attribute 'version'
Fontconfig warning: line 9: unknown element "description"
Fontconfig error: Cannot load default config file
000f:err:service:process_send_command receiving command result timed out
000f:fixme:service:scmdatabase_autostart_services Auto-start service L"WineBus" failed to start: 1053
Fontconfig warning: line 5: unknown element "its:rules"
Fontconfig warning: line 6: unknown element "its:translateRule"
Fontconfig error: line 6: invalid attribute 'translate'
Fontconfig error: line 6: invalid attribute 'selector'
Fontconfig error: line 7: invalid attribute 'xmlns:its'
Fontconfig error: line 7: invalid attribute 'version'
Fontconfig warning: line 9: unknown element "description"
Fontconfig error: Cannot load default config file
Fontconfig warning: line 5: unknown element "its:rules"
Fontconfig warning: line 6: unknown element "its:translateRule"
Fontconfig error: line 6: invalid attribute 'translate'
Fontconfig error: line 6: invalid attribute 'selector'
Fontconfig error: line 7: invalid attribute 'xmlns:its'
Fontconfig error: line 7: invalid attribute 'version'
Fontconfig warning: line 9: unknown element "description"
Fontconfig error: Cannot load default config file
Fontconfig warning: line 5: unknown element "its:rules"
Fontconfig warning: line 6: unknown element "its:translateRule"
Fontconfig error: line 6: invalid attribute 'translate'
Fontconfig error: line 6: invalid attribute 'selector'
Fontconfig error: line 7: invalid attribute 'xmlns:its'
Fontconfig error: line 7: invalid attribute 'version'
Fontconfig warning: line 9: unknown element "description"
Fontconfig error: Cannot load default config file
0027:err:winediag:nodrv_CreateWindow Application tried to create a window, but no driver could be loaded.
0027:err:winediag:nodrv_CreateWindow The explorer process failed to start.
0027:err:ole:apartment_createwindowifneeded CreateWindow failed with error 0
0027:err:ole:apartment_createwindowifneeded CreateWindow failed with error 0
0027:err:ole:apartment_createwindowifneeded CreateWindow failed with error 0
Fontconfig warning: line 5: unknown element "its:rules"
Fontconfig warning: line 6: unknown element "its:translateRule"
Fontconfig error: line 6: invalid attribute 'translate'
Fontconfig error: line 6: invalid attribute 'selector'
Fontconfig error: line 7: invalid attribute 'xmlns:its'
Fontconfig error: line 7: invalid attribute 'version'
Fontconfig warning: line 9: unk
"

**Comment by hrotkogabor** [comment]
After I got the same symptoms, I installed the game with PlayOnLinux. With that, I got 5 sec startup.

**Comment by michas0602** [comment]
Same here (?) - just a bluescreen on startup. 
System is Ubuntu Mate 20.04 daily with Nvidia-440-driver.

**Comment by mbretter** [comment]
same here ubuntu 20.04.1:
snap    2.45.3.1
snapd   2.45.3.1
series  16
ubuntu  20.04
kernel  5.8.0-050800-generic

**Comment by bothamad** [comment]
Finally i have an answer i was looking at this and went wtf does a font config file have to do with wine ,but like whatever tickles your fancy!!

SO THE MOMENT YOUVE ALL BEEN WAITING FOR YOU JUST NEED TO ROLLBACK THE FONT-CONFIG PACKAGE?!

So yeah here are the instructions hahahah :)


The answer @randomlogin by worked for me, but I can't list the detailed commands I used as a comment to help clarify what to actually do.

So, as an answer with extra detail instead of a comment, here is what worked for me:

    Go to 

https://packages.ubuntu.com/bionic/all/fontconfig-config/download and download the .deb file

    Check the .deb file downloaded correctly comparing to the download page's MD5 sum: 
md5sum ~/Downloads/fontconfig-config_2.12.6-0ubuntu2_all.deb

    "Manually" install the .deb file: 
sudo dpkg --install ~/Downloads/fontconfig-config_2.12.6-0ubuntu2_all.deb

    Copy fontconfig files: 
cp -L -r /etc/fonts/conf.d /tmp/etc-fonts-conf.d

   run
sudo apt --fix-broken install

    Copy back the fontconfig files 
sudo  cp -L -r  /tmp/etc-fonts-conf.d/* /etc/fonts/conf.d/

    Edit /etc/fonts/fonts.conf and remove the first few lines that start with the tag <its: and the tag <description>

Credit to @RandomLogin and @a5kin over on askubuntu.com :)

**Comment by hyphone** [comment]
Isn't the purpose of snaps not to do steps like this and potentially mess up your system? I would not consider this as a fixAm 07.10.2020 14:08 schrieb bothamad <notifications@github.com>:
Finally i have an answer i was looking at this and went wtf does a font config file have to do with wine ,but like whatever tickles your fancy!!
SO THE MOMENT YOUVE ALL BEEN WAITING FOR YOU JUST NEED TO ROLLBACK THE FONT-CONFIG PACKAGE?!
So yeah here are the instructions hahahah :)
The answer @randomlogin by worked for me, but I can't list the detailed commands I used as a comment to help clarify what to actually do.
So, as an answer with extra detail instead of a comment, here is what worked for me:
Go to 

https://packages.ubuntu.com/bionic/all/fontconfig-config/download and download the .deb file
Check the .deb file downloaded correctly comparing to the download page's MD5 sum: 

md5sum ~/Downloads/fontconfig-config_2.12.6-0ubuntu2_all.deb
"Manually" install the .deb file: 

sudo dpkg --install ~/Downloads/fontconfig-config_2.12.6-0ubuntu2_all.deb
Copy fontconfig files: 

cp -L -r /etc/fonts/conf.d /tmp/etc-fonts-conf.d
run
sudo apt --fix-broken install
Copy back the fontconfig files 

sudo  cp -L -r  /tmp/etc-fonts-conf.d/* /etc/fonts/conf.d/
Edit /etc/fonts/fonts.conf and remove the first few lines that start with the tag <its: and the tag <description>

Credit to @randomlogin and @a5kin over on askubuntu.com :)

**Comment by xordspar0** [comment]
> Finally i have an answer

Woo, nice work!

> Isn't the purpose of snaps not to do steps like this and potentially mess up your system? I would not consider this as a fix

You're right, it's not a fix *per se*, but it's something we can work with. It's been a while, but we might get this snap working again. :slightly_smiling_face:

**Comment by Rav317** [comment]
Thanks, worked for me.

**Comment by xordspar0** [comment]
Now that https://github.com/snapcrafters/tmnationsforever/pull/30 is merged, the project is restructured significantly. I can play the game without the workaround posted above. I think this issue should be closed now.

**Comment by Martincic** [comment]
Still happens every X runs. I get this after opening/closing the game between 5-10 times and then I have to re-install the thing.

**Comment by xordspar0** [comment]
@Martincic, I think you're experiencing a new issue. This issue is about a failure in the install process that made it impossible to ever start the game.
